### DISCUSSION TITLE
ERROR - registra tu solución en línea

### DISCUSSION THREAD

**Discussion by MariaDelgado17** [rramosp/ai4eng.v1#53] [00 - General]
Buenas tardes,

Ejecuto esta linea de código:
"student.submit_task(globals(), task_id="task_01");"
 y me dá el sigueinte error:

---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-14-4cdaf622784a> in <module>
----> 1 student.submit_task(globals(), task_id="task_01");

NameError: name 'student' is not defined

**Comment by MariaDelgado17** [comment]
Buenas tardes, me pueden ayudar con mi error? 

**Gracias**

**Comment by Felipe-RA** [comment]
Saludos MariaDelgado

Disculpa, que no te había visto.

Este error suele ocurrir al no hacer login o hacer un login fallido.

Verifica que estes logeandote correctamente, si sigues con problemas vuelve a escribirme

Hasta pronto

**Reply by MariaDelgado17** [reply]
Muchas gracias, Felipe, ya reviso y te confirmo!!

**Reply by MariaDelgado17** [reply]
Muchas gracias Felipe..mi error era que habia cambiado el varname.

**_Saludos_**
